index.html

```diff
@@ -40,5 +44,191 @@ <h4>LET’S CONNECT</h4>
+            <div class="article_text">
+                <h2 class="product-title">Facebook 360</h2>
+                <ul class="type_dev">
+                    <li class="li-color">FACEBOOK</li>
+                    <li><img src="assets/images/Counter.svg" alt="Counter image"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="assets/images/Counter.svg" alt="Counter image"></li>
+                    <li>2015</li>
+                </ul>
+                <p class="product-details">
+                    Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and
+                    videos on Gear VR.
+                </p>
+                <ul class="product-categories">
+                    <li><button class="button-category">HTML</button></li>
+                    <li><button class="button-category">CSS</button></li>
+                    <li><button class="button-category">JavaScript</button></li>
+                </ul>
+                <button class="button-project">See project</button>
+            </div>
+
+        </article>
+        <article class="div-works div-works-reverse ">
+            <img class="article_img" src="assets/images/Snapshoot Portfolio.png" alt="">
+            <div class="article_text">
+                <h2 class="product-title">Uber Navigation</h2>
+                <ul class="type_dev">
+                    <li class="li-color">Uber</li>
+                    <li><img src="assets/images/Counter.svg" alt="Counter image"></li>
+                    <li>Lead Developer</li>
+                    <li><img src="assets/images/Counter.svg" alt="Counter image"></li>
+                    <li>2015</li>
+                </ul>
+                <p class="product-details">
+                    A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+                </p>
+                <ul class="product-categories">
+                    <li><button class="button-category">html</button></li>
+                    <li><button class="button-category">Ruby on rails</button></li>
+                    <li><button class="button-category">css</button></li>
+                    <li><button class="button-category">javaScript</button></li>
+                </ul>
+                <button class="button-project">See project</button>
+            </div>
+        
+        </article>
+
+
+    </section>
+    <section class="about-my-self main-section" id="about-my-self main-section">
+        <div class="div-description-about">
+            <h2 class="about-header">About myself</h2>
+            <p class="about-text">Hello I’m a software developer! I can help you build a product , feature or website.
+                Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
```

Comment:
Well done on implementing the buttons and meeting the other requirements! 👍 However, there are a couple of additional changes that need to be made based on the previous reviewer's request:

- [ ] The "LET'S CONNECT" button should have the color `#7F8CFF` as per the design. Please update the button's color accordingly.
- [ ] Additionally, the paragraph in the about section should have the color `#344563` as requested by the previous reviewer. Please ensure that the paragraph's color matches the design.

By addressing these changes, you will bring the elements in line with the design specifications. Keep up the great work! 🎉

Reply:
Thanks a lot dear @shahadat3669